test/MarketMaker.js

```diff
@@ -254,7 +236,30 @@ contract('marketMaker', function(accounts) {
     }
   });
 
-  it.only("should go through scenario with some claims and sells on the Market Maker", async function() {
+  var checkScenarioProperties = async function(data, mm, customer) {
```

Comment:
good, we needed to move this block long ago. just one thing @AugustoL: next time you move code, try not to change it in the same commit, it makes it way harder to see what has changed. Try to do it in separate commits.